JDBC/OAuthSampleApp/src/main/java/OAuthSampleApp.java

```diff
@@ -92,108 +92,101 @@ private static void SetUpDbForOAuth() throws Exception
 		st.execute("GRANT AUTHENTICATION v_oauth TO " + USER + ";");
 		st.execute("GRANT ALL ON SCHEMA PUBLIC TO " + USER +";");
 		st.close();
-    }
-    
-    
-    private static void TearDown() throws Exception
-    {
-		System.out.println("TearDown");
-    	Statement st = vConnection.createStatement(); 
-    	String USER = prop.getProperty("User");
-    //    st.execute("DROP USER IF EXISTS " + USER + " CASCADE");
-    //    st.execute("DROP AUTHENTICATION IF EXISTS v_oauth CASCADE");
-        vConnection.close();
-    }
-
+	}
+	// Dispose the authentication record from database
+	private static void TearDown() throws Exception 
+	{
+		Statement st = vConnection.createStatement(); 
+		String USER = prop.getProperty("User");
+		st.execute("DROP USER IF EXISTS " + USER + " CASCADE");
+		st.execute("DROP AUTHENTICATION IF EXISTS v_oauth CASCADE");
+		vConnection.close();
+	}
+	// Connect to Database using access Token
 	private static Connection connectToDB(String accessToken, String clientId, String clientSecret, String tokenUrl) throws SQLException 
-    {
-        Properties jdbcOptions = new Properties();
-        jdbcOptions.put("oauthaccesstoken", accessToken);
-        jdbcOptions.put("oauthtokenurl", tokenUrl);
-        jdbcOptions.put("oauthclientid", clientId);
-        jdbcOptions.put("oauthclientsecret", clientSecret);
-
+	{
+		Properties jdbcOptions = new Properties();
+		jdbcOptions.put("oauthaccesstoken", accessToken);
+		jdbcOptions.put("oauthtokenurl", tokenUrl);
+		jdbcOptions.put("oauthclientid", clientId);
+		jdbcOptions.put("oauthclientsecret", clientSecret);
 		return DriverManager.getConnection(
-                "jdbc:vertica://" + "172.17.0.45" + ":" + "5433" + "/" + "verticadb21451", jdbcOptions);
+				"jdbc:vertica://" + getParam("Host") + ":" + getParam("Port") + "/" + getParam("Database") , jdbcOptions);
     }
-
-    private static void ConnectToDatabase() throws SQLException {
-		System.out.println("ConnectToDatabase");
+	// Test connection using access token and test database query result
+	private static void ConnectToDatabase() throws SQLException 
+	{
 		String accessToken = System.getProperty(OAUTH_ACCESS_TOKEN_VAR_STRING);
-
 		if( null == accessToken || accessToken.isEmpty())
 		{
 			System.out.println("Access Token is not available.");
 		}else
 		{
-			Connection conn = connectToDB( accessToken,
-										prop.getProperty("ClientId"),
-										prop.getProperty("ClientSecret"),
-										prop.getProperty("TokenUrl"));
+			Connection conn = connectToDB(	accessToken,
+											prop.getProperty("ClientId"),
+											prop.getProperty("ClientSecret"),
+											prop.getProperty("TokenUrl"));
```

Comment:
The properties aside from the access token shouldn't be needed.

Reply:
Removed other properties.